client/server_scripts/install_docker.sh

```diff
@@ -1,18 +1,19 @@
-if which apt-get > /dev/null 2>&1; then pm=$(which apt-get); docker_pkg="docker.io"; dist="debian";\
-elif which dnf > /dev/null 2>&1; then pm=$(which dnf); docker_pkg="docker"; dist="fedora";\
-elif which yum > /dev/null 2>&1; then pm=$(which yum); docker_pkg="docker"; dist="centos";\
+if which apt-get > /dev/null 2>&1; then pm=$(which apt-get); silent_inst="install -yq"; silent_up="update -yq"; docker_pkg="docker.io"; dist="debian";\
+elif which dnf > /dev/null 2>&1; then pm=$(which dnf); silent_inst="install -yq"; silent_up="update -yq"; docker_pkg="docker"; dist="fedora";\
+elif which yum > /dev/null 2>&1; then pm=$(which yum); silent_inst="install -y -q"; silent_up="update -y -q"; docker_pkg="docker"; dist="centos";\
 else echo "Packet manager not found"; exit 1; fi;\
-echo "Dist: $dist, Packet manager: $pm, Docker pkg: $docker_pkg";\
+echo "Dist: $dist, Packet manager: $pm, Install command: $silent_inst, Update command: $silent_up,  Docker pkg: $docker_pkg";\
 if [ "$dist" = "debian" ]; then export DEBIAN_FRONTEND=noninteractive; fi;\
-if ! command -v sudo > /dev/null 2>&1; then $pm update -yq; $pm install -yq sudo; fi;\
-if ! command -v fuser > /dev/null 2>&1; then sudo $pm install -yq psmisc; fi;\
-if ! command -v lsof > /dev/null 2>&1; then sudo $pm install -yq lsof; fi;\
-if ! command -v docker > /dev/null 2>&1; then sudo $pm update -yq; sudo $pm install -yq $docker_pkg;\
+if ! command -v sudo > /dev/null 2>&1; then $pm $silent_up; $pm $silent_inst sudo;\
+elif ! command -v fuser > /dev/null 2>&1 || ! command -v lsof > /dev/null 2>&1 || ! command -v docker > /dev/null 2>&1; then sudo $pm $silent_up; fi;\
+if ! command -v fuser > /dev/null 2>&1; then sudo $pm $silent_inst psmisc; fi;\
+if ! command -v lsof > /dev/null 2>&1; then sudo $pm $silent_inst lsof; fi;\
+if ! command -v docker > /dev/null 2>&1; then sudo $pm $silent_inst $docker_pkg;\
   if [ "$dist" = "fedora" ] || [ "$dist" = "centos" ] || [ "$dist" = "debian" ]; then sudo systemctl enable docker && sudo systemctl start docker; fi;\
 fi;\
 if [ "$dist" = "debian" ]; then \
   docker_service=$(systemctl list-units --full --all | grep docker.service | grep -v inactive | grep -v dead | grep -v failed);\
-  if [ -z "$docker_service" ]; then sudo $pm update -yq; sudo $pm install -yq curl $docker_pkg; fi;\
+  if [ -z "$docker_service" ]; then sudo $pm $silent_up; sudo $pm $silent_inst curl $docker_pkg; fi;\
   sleep 3 && sudo systemctl start docker && sleep 3;\
 fi;\
 if ! command -v sudo > /dev/null 2>&1; then echo "Failed to install Docker";exit 1;fi;\
```

Comment:
Why command does not match error message? May be last check should be for docker, not sudo?

Reply:
Minimal changes were made to this part of the script: only replacing commands with variables.
To minimize the likelihood of new errors appearing.
This part of the script should be analyzed and reworked in the next logical group of changes related to checking the status of docker.service for all values ​​of the $dist variable, in cases where, before using AmneziaVPN, docker was installed on the server, but for some reason, it is inactive or does not start automatically when the server boots while this script is running.

Reply:
Out of scope for the current PR, probably should be fixed further.